ttnn/cpp/ttnn/operations/full_like/device/kernels/writer_full_like.cpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include <cstdint>
+#include "dataflow_api.h"
+#include "debug/dprint.h"
+
+void kernel_main() {
+    std::uint32_t output_addr = get_arg_val<uint32_t>(0);
+    std::uint32_t num_tiles = get_arg_val<uint32_t>(1);
+    std::uint32_t start_id = get_arg_val<uint32_t>(2);
+    std::uint32_t fill_value = get_arg_val<uint32_t>(3);
+
+    constexpr uint32_t cb_value = tt::CB::c_intermed0;
+    const uint32_t cb_page_size = get_tile_size(cb_value);
+    const auto cb_data_format = get_dataformat(cb_value);
+    const InterleavedAddrGenFast<true> s = {
+        .bank_base_address = output_addr, .page_size = cb_page_size, .data_format = cb_data_format};
+
+    cb_wait_front(cb_value, 1);
```

Comment:
Use one_tile variable like you did in reader.

Reply:
Resolved